Testreglar/3.3.3/App/app-3.3.3a.json

```diff
@@ -0,0 +1,233 @@
+			"ht": "<p>Beskriv skjema/skjemaelement, slik at det er mulig å identifisere det i ettertid. </p>\n<p><strong>Merk:</strong> Hvis skjema/skjemaelement inngår i en prosess skal du også beskrive prosessen.</p>",
+			"type": "tekst",
+			"label": "Skjema/prosess:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Oppdager skjemaet automatisk at utfylt inndata har feil format eller verdi?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Legg inn feil inndata i alle skjemaelement.\n<ul>\n<li>Hvis skjemaelementer er forhåndsutfylt, endre forhåndsutfylt informasjon til feil inndata format eller verdi der det er mulig.</li>\n<li>Forhåndsutfylt informasjon betyr at det allerede står inndata i skjemaet når appsiden er lastet inn.</li>\n<li>I sitasjon appsiden krever informasjon, men som brukeren har utelatt, for eksempel du ikke har krysset av i en avkryssingsboks</li>\n</ul>\n</li>\n<li>Prøv deretter å fullføre/sende inn eller gå videre i skjemaet.</li>\n</ul>\n<p><strong>Merk:</strong> Skjema der knappen for å gå videre eller fullføre er deaktivert, skal testes.</p>\n<p>Et deaktivert skjemaelement for å gå videre eller fullføre skjemaet, som blir aktivt når brukeren fyller ut informasjon rett format eller verdi, indikerer at brukeren har fylt ut inndata som faller utenfor påkrevd dataformat eller verdier, og at dette oppdages automatisk, uten at brukeren er involvert.</p>\n<p>Eksempel på feil inndata er:</p>\n<ul>\n<li>Postnummer som inneholder bokstaver.</li>\n<li>Navn som inneholder tall eller spesialtegn.</li>\n<li>Ugyldig e-postformat.</li>\n<li>Ugyldig datoformat.</li>\n<li>Ugyldig telefonnummer.</li>\n<li>Data utenfor grenseverdi, for eksempel måned 14 i en dato. </li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Skjema oppdager ikke automatisk at utfylt inndata har feil format eller verdi.",
+					"element": "2.3"
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilket skjemaelement tester du?",
+			"ht": "<p>Beskriv elementet, slik at det er mulig å identifisere det i ettertid. Hvis det gjelder flere elementer, registrerer du et og et.</p>",
+			"type": "tekst",
+			"label": "Skjemaelement:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Blir informasjonen du har fylt ut liggende i skjemaelementet?",
+			"ht": "<p>Hovedregelen er at informasjon som er fylt ut skal bli liggende i skjemaet, selv om det blir oppdaget inndatafeil. Unntaket er dersom informasjonen er knyttet til personvern eller sikkerhet (sensitiv informasjon). </p>\n<p><strong>Merk:</strong> Det er ok at feil inndata blir korrigert automatisk.</p>",
```

Comment:
Teknikken sier vel ingenting om at ting knyttet til personvern er unntatt. Videre så er det ok at feil inndata blir korrigert automatisk men det skal fortsatt stå at det har hatt en feil osv.

Reply:
Fjernet presisering personvern for å unngå misforståelse.